client/reader/stream/style.scss

```diff
@@ -661,6 +661,19 @@
 			overflow: hidden;
 			-webkit-line-clamp: 1;
 			-webkit-box-orient: vertical;
+			white-space: nowrap;
+			width: 225px;
+			position: relative;
+
+			&::after {
+				content: "";
```

Comment:
I discovered we have a mixin for this, if you'd like to use it. Try `@include long-content-fade( $size: 5% );`

The file is here if you'd like to see the underlying styles: https://github.com/Automattic/wp-calypso/blob/70949b8a24f42a187be437e9430a65891abc3e4b/client/assets/stylesheets/shared/mixins/_long-content-fade.scss

Reply:
Good eye! I made this update here: https://github.com/Automattic/wp-calypso/pull/78858/commits/1e0fdd182eea8da14f34f0f6926a0a3fd35271fe